uint8 TURN_BEGIN = 0
uint8 BUFF_ATK = 1
uint8 BUFF_DEF = 2
uint8 BUFF_HEALTH = 3
uint8 BUFF_CLEAR = 4
uint8 HARM = 5
uint8 BLOCKED = 6
uint8 GAME_OVER = 7
uint8 cmd

int32 id    # id = -1 to broadcast to all robots

float64 buff_value
float64 harm_value